packages/client/src/arpa_reporter/views/NewUploadView.vue

```diff
@@ -79,7 +79,7 @@ export default {
               name: 'submit',
               type: 'submit',
               label: 'Submit',
-              inputClass: ['btn', 'btn-primary'],
+              inputClass: ['btn', 'usdr-btn-primary'],
```

Comment:
Note to reviewer:  Screenshots

## Before

![image](https://github.com/user-attachments/assets/45fc299b-63d4-49ea-8812-ca9c8d6df1e7)


## After

![image](https://github.com/user-attachments/assets/dea4f36a-265f-4839-9343-73805ed0fd6e)